src/main/webapp/panelStyle.js

```diff
@@ -0,0 +1,174 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* eslint-disable no-unused-vars */
+
+// Script for handling the behaviour of the panels' layout
+
+// Add appropriate event listeners to all toggle icons
+document.querySelectorAll('.panel__toggle-icon')
+    .forEach((element) => {
+      switch (element.id) {
+        case 'assignSettingsIcon':
+          element
+              .addEventListener(
+                  'click',
+                  (event) => displayAssignSettings(event)
+              );
+          break;
+        default:
+          element
+              .addEventListener(
+                  'click',
+                  (event) => toggleIconHandler(event)
+              );
+      }
+    });
+
+/**
+ * Changes icon from clicked to un-clicked, and vice versa.
+ * Assumes parent div has two img elements, one hidden and one not.
+ *
+ * @param {Event} event event generated from click
+ */
+function toggleIconHandler(event) {
+  const currentIcon = event.target;
+  const currentDiv = currentIcon.parentNode;
+  const newIcon = currentDiv.querySelector('img[hidden]');
+
+  currentIcon.setAttribute('hidden', '');
+  newIcon.removeAttribute('hidden');
+}
+
+/**
+ * Shows/hides the settings display in the assign panel
+ *
+ * @param {Event} event event generated from click
+ */
+function displayAssignSettings(event) {
+  // Change icon appearance
+  toggleIconHandler(event);
+
+  // TODO: show/hide the settings display
```

Comment:
Is this the same as the one in assignPanelStyle.js?

Reply:
Fixed!